### PR TITLE
Portal bugfixes

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

I spent last night on testing portal to find all bugs and improve overall implementation.
I found so many bugs that I was almost impossible to get them all logged into jira - especially that fixes were pretty simple. Decided to split this work into separate commits to simplify verification.
After this change portal seems to be working properly. 

@paolobueno - had problem with finding fix for data not updating after edit.
I couldn't figure out in reasonable time how to update cached model after edit.
After disabling http cache in http module problem seems to disappear. 
If you are ok with that we can leave it this way. If not we can create separate ticket to enable cache, but still make sure that models are up to date.

Note: Improvements to WFM module will be done as part of the separate PR (core)

See each commit for verification.

### CODE REVIEW

File: packages/auth-passport/authService.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ var cloudUrl;
 var dialog;
```
Comment (paolobueno) [inline packages/auth-passport/authService.js]: looks like you've replaced this variable with a class member, just need to remove it

**Comment by paolobueno** [comment]
@wtrocki do you want to rename the folders as well to match the new package names?

**Comment by wtrocki** [comment]
@paolobueno We can and I wanted to do that, but I did not want to introduce so many changes in this PR as it may be not easy to verify later. I will do rename as part of separate PR if you do not mind. Renames will introduce more than 80 files changed.

File: packages/http/configCache.js [hunk 2]
```diff
@@ -1,4 +0,0 @@
-angular.module('wfm.http').config(['$httpProvider',
-  function($httpProvider) {
-    $httpProvider.defaults.cache = true;
-  }]);
```
Comment (paolobueno) [inline packages/http/configCache.js:4]: :+1: for removing this, can be configured in the app level if required.

The caching seems to be too aggressive, hard to pick up updates. express already handles 304 responses nicely, so no need to for aggressive client caching in the portal.

**Review by paolobueno — DISMISSED** [review]
Verified locally

**Review by paolobueno — APPROVED** [review]

**Pull request merged**